tests/configure.test.ts

```diff
@@ -48,6 +48,7 @@ const removeAppDir = async (folder: string) =>
 describe('configure', () => {
   describe('default', () => {
     beforeAll(async () => {
+      await removeAppDir(appFolder);
```

Comment:
Depending on when you interrupt these test, you may end up with a dirty state the next time you run the test. This ensures the test directory is always clean.